src/main/java/racingcar/model/Game.java

```diff
@@ -0,0 +1,69 @@
+package racingcar.model;
+
+import static racingcar.constant.Constant.END_CONDITION;
+import static racingcar.constant.Constant.INIT_MOVE;
+import static racingcar.constant.Constant.MAX_RANDOM_NUMBER;
+import static racingcar.constant.Constant.MIN_RANDOM_NUMBER;
+import static racingcar.constant.Constant.MOVABLE_RANDOM_NUMBER;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+    private final Car cars;
+    private int remainingSet;
+    private int maxMove;
+
+    public Game(Car cars, String set) {
+        this.cars = cars;
+        this.remainingSet = Integer.parseInt(set);
+        maxMove = INIT_MOVE;
+    }
+
+    public boolean isContinue() {
+        return remainingSet > END_CONDITION;
+    }
+
+    public void playSet() {
+        for (String carName : cars.getCarNameSet()) {
+            int distance = cars.getMove(carName);
+
+            if (canGo()) {
+                cars.updateMove(carName, distance + 1);
+
+                updateMaxMove(carName);
+            }
+        }
+
+        remainingSet--;
+    }
+
+    private boolean canGo() {
+        int randomNumber = Randoms.pickNumberInRange(MIN_RANDOM_NUMBER, MAX_RANDOM_NUMBER);
+
+        return randomNumber >= MOVABLE_RANDOM_NUMBER;
+    }
+
+    private void updateMaxMove(String carName) {
+        if (cars.getMove(carName) > maxMove) {
+            maxMove = cars.getMove(carName);
+        }
+    }
+
+    public Car getCars() {
+        return cars;
+    }
+
+    public List<String> getWinnerList() {
+        List<String> winner = new ArrayList<>();
+
+        for (String carName : cars.getCarNameSet()) {
+            if (cars.getMove(carName) == maxMove) {
+                winner.add(carName);
+            }
+        }
+
+        return winner;
+    }
```

Comment:
```suggestion
    public List<String> getWinnerList() {
       return cars.getCarNameSet()
              .stream()
              .filter(carName -> cars.getMove(carName) == maxPosition)
              .collect(Collectors.toList());
    }
```
컬렉션으로 바꾼다면 위와 같이 바꿀 수 있을 것 같아요! 소윤님은 어떤게 더 가독성 있다고 느껴지나요??
더 가독성 있다고 느껴지는 방식을 활용하시면 좋을 것 같습니다!